README.md

```diff
@@ -5,7 +5,7 @@
 This repository contains the documentation and configuration necessary to run a 
 Raiden Transport Matrix server.
 
-**Current release:** [2018.12.0](https://github.com/raiden-network/raiden-transport/tree/2018.12.0)
+**Current release:** [2019.10.1](https://github.com/raiden-network/raiden-transport/tree/2019.10.1)
```

Comment:
There's 2 further changes necessary in https://github.com/raiden-network/raiden-transport/pull/37/files#diff-04c6e90faac2675aa89e2176d2eec7d8R134-R138

Reply:
thanks for your eyes!!